lib/theme_template/theme.gemspec.erb

```diff
@@ -10,9 +11,7 @@ Gem::Specification.new do |spec|
   spec.homepage      = "TODO: Put your gem's website or public repo URL here."
   spec.license       = "MIT"
 
-  spec.files         = `git ls-files -z`.split("\x0").select { |f| f.match(%r{^(<%= theme_directories.join("|") %>|LICENSE|README)/i}) }
+  spec.files         = `git ls-files -z`.split("\x0").grep(Jekyll::ThemeBuilder::GEM_INCLUSION_REGEX)
 
   spec.add_development_dependency "jekyll", "~> <%= jekyll_version_with_minor %>"
-  spec.add_development_dependency "bundler", "~> 1.12"
-  spec.add_development_dependency "rake", "~> 10.0"
```

Comment:
These are required. Why did you remove these? The release process is `bundle exec rake release`.